README.md

```diff
@@ -28,14 +28,26 @@ which is located in `tests/conftest.py`.
 # Setup
 Create and activate your conda environment with python3.9:
 ```commandline
-conda create -y -n substrait_consumer_testing -c conda-forge python=3.9
+conda create -y -n substrait_consumer_testing -c conda-forge python=3.9 openjdk
 conda activate substrait_consumer_testing
 ```
+*Note: Java is used by Jpype to access the Isthmus producer.  
+JPype has been tested with Java versions from Java 1.7 to Java 13.
```

Comment:
This seems like the kind of thing that will go stale quickly.  Maybe something like "JPype should work with all versions of Java but to see details on which versions are officially supported see https://jpype.readthedocs.io/en/latest/install.html"